### PR TITLE
Reintroduce requirements install in setup

### PR DESCRIPTION
**Pull request by northwestwitch**
Installed requirements.txt in the setup

**How to test**:
1. `pip install scout-browser`  in a clean environment and make sure that it runs by executing any cli command

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [ ] tests executed by

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ If you would like to install Scout for local development: -->
 ```bash
 git clone https://github.com/Clinical-Genomics/scout
 cd scout
-pip install --requirement requirements.txt --editable .
```
Comment (dnil) [inline README.md:60]: :+1:

**Review by moonso — APPROVED** [review]
👍

**Review by dnil — APPROVED** [review]
Looks good to me!

**Comment by northwestwitch** [comment]
Thanks, let's merge as soon as it passes the tests

**Pull request merged**